packages/rect/index.d.ts

```diff
@@ -0,0 +1,24 @@
+declare module "@reach/rect" {
+  import { Ref } from "react"
+  interface ClientRect {
```

Comment:
Another little thing I noticed is that there's a type definition built in for `DOMRect` which you could use here instead of re-defining it. Here's a link to the [DOMRect definition](https://github.com/Microsoft/TypeScript/blob/master/src/lib/dom.generated.d.ts#L3801). You should be able to use it here by adding `/// <reference lib="dom"/>` to the top of this definition file. Here's a link to the [triple slash documentation](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html)